### PR TITLE
cracen: Platform keys support

### PR DESCRIPTION
**Pull request by vili-nordic**
Support for keys that are embedded into FW.

### CODE REVIEW

**Review by Vge0rge — APPROVED** [review]

File: subsys/nrf_security/src/drivers/cracen/cracenpsa/src/platform_keys/platform_keys.c [hunk 1]
```diff
@@ -60,130 +80,368 @@ typedef struct sicr_key {
...
+	{0x4000BB01,
+		{
+#include <public_key_native_MANIFEST_PUBKEY_NRF_TOP_1.bin.inc>
+		},
+	 32,
+	 PSA_KEY_TYPE_ECC_PUBLIC_KEY(PSA_ECC_FAMILY_TWISTED_EDWARDS),
+	 255},
+	{0x4000BB02,
+		{
+#include <public_key_native_MANIFEST_PUBKEY_NRF_TOP_1.bin.inc>
```
Comment (tomchy) [inline subsys/nrf_security/src/drivers/cracen/cracenpsa/src/platform_keys/platform_keys.c:108]: Should not that be `public_key_native_MANIFEST_PUBKEY_NRF_TOP_2`?

File: subsys/nrf_security/src/drivers/cracen/cracenpsa/src/platform_keys/platform_keys.c [hunk 2]
```diff
@@ -60,130 +80,368 @@ typedef struct sicr_key {
...
+		},
+	 32,
+	 PSA_KEY_TYPE_ECC_PUBLIC_KEY(PSA_ECC_FAMILY_TWISTED_EDWARDS),
+	 255},
+};
```
Comment (tomchy) [inline subsys/nrf_security/src/drivers/cracen/cracenpsa/src/platform_keys/platform_keys.c:113]: What about `public_key_native_MANIFEST_PUBKEY_SYSCTRL_[0-2]` keys?

**Review by tomchy — CHANGES_REQUESTED** [review]

File: subsys/nrf_security/src/drivers/cracen/cracenpsa/src/platform_keys/platform_keys.c (hunk 1 above)
Comment (vili-nordic) [reply subsys/nrf_security/src/drivers/cracen/cracenpsa/src/platform_keys/platform_keys.c:108]: will fix

File: subsys/nrf_security/src/drivers/cracen/cracenpsa/src/platform_keys/platform_keys.c (hunk 2 above)
Comment (vili-nordic) [reply subsys/nrf_security/src/drivers/cracen/cracenpsa/src/platform_keys/platform_keys.c:113]: I'll add them

**Review by tomchy — APPROVED** [review]

**Pull request merged**